clang/lib/CodeGen/CGCall.cpp

```diff
@@ -199,15 +199,17 @@ static void appendParameterTypes(const CodeGenTypes &CGT,
                               prefix.size());
 }
 
+using SmallExtParameterInfoList =
```

Comment:
I would suggest skipping `Small` in each of these.

ALSO I find myself wondering what work we did to justify 16 for each of these, thats a fairly sizable number, that I cannot help but wonder if comes from a time when there was no default.

Not something particularly actionable here (unless you want to try a change and spawn the compile-time-tracker), but something I note.

Reply:
Done; I dropped the `Small` prefix.

With regard to the in-place size of 16, I'm guessing that was a spur of the moment pick that "felt right" way back when. I did look for other aliases of `SmallVector` for these types and that in-place size and didn't find any. There is a case of `SmallVector<CanQualType, 2>` elsewhere in the file, so it wasn't a blind default everywhere.